src/main/java/ch/njol/skript/util/Utils.java

```diff
@@ -714,5 +715,37 @@ public static long parseLong(final String s) {
 		}
 		return lastIndex;
 	}
+
+	private static void checkSpawnerTypes() { // If this was in static load section it would fail due to MC version field not yet initialized
+		if (!spawnerTypes.isEmpty())
+			return;
+
+		for (org.bukkit.entity.EntityType e : org.bukkit.entity.EntityType.values()) {
+			Class<? extends Entity> c = e.getEntityClass();
+			if (c != null)
+				spawnerTypes.put(EntityData.fromClass(c), e); // Cache Skript EntityData -> Bukkit EntityType
+		}
+	}
+
+	/**
+	 * Convert from Skript's EntityData to Bukkit's EntityType
+	 * @param e Skript's EntityData
+	 * @return Bukkit's EntityType
+	 */
+	public static org.bukkit.entity.EntityType toBukkitEntityType(EntityData e){
```

Comment:
```suggestion
	public static org.bukkit.entity.EntityType toBukkitEntityType(EntityData e) {
```
same for method below